pkg/build/nodeimage/internal/kube/builder_bazel.go

```diff
@@ -81,42 +82,24 @@ func (b *bazelBuilder) Build() (Bits, error) {
 		return nil, err
 	}
 
-	// https://docs.bazel.build/versions/master/output_directories.html
-	binDir := filepath.Join(b.kubeRoot, "bazel-bin")
-	buildDir := filepath.Join(binDir, "build")
-	bazelGoosGoarch := fmt.Sprintf("linux_%s", b.arch)
-
-	// helpers to get the binary paths which may or may not be "pure" (no cgo)
-	// Except for kubelet, these are pure in Kubernetes 1.14+
-	// kubelet is at bazel-bin/cmd/kubelet/kubelet since 1.14+
-	// TODO: do we care about building 1.13 from source once we add support
-	// for building from release binaries?
-	// https://github.com/kubernetes/kubernetes/pull/73930
-	strippedCommandPath := func(command string) string {
-		return filepath.Join(
-			binDir, "cmd", command,
-			fmt.Sprintf("%s_stripped", bazelGoosGoarch), command,
-		)
+	kubeletPath, err := findGoBinary("//cmd/kubelet:kubelet")
+	if err != nil {
+		return nil, fmt.Errorf("could not find kubelet: %w", err)
```

Comment:
kind is using a custom package for errors:
https://github.com/kubernetes-sigs/kind/tree/master/pkg/errors/errors.go

e.g. the above should be `errors.Wrap(err, "could not find kubelet")`

Reply:
Fixed.